style.css

```diff
@@ -655,6 +778,73 @@ form > textarea {
     color: #fff;
   }
 
+  .btn-popup {
+    padding: 8px;
+    border-radius: 5px;
+    margin: 8px 0;
+    transition: all 0.5s;
+    border: 2px solid #7f8cff;
+    color: #6070ff;
+    font-size: 18px;
+    font-weight: 500;
+    background-color: #fff;
+  }
+
+  .popup {
+    position: fixed;
+    top: 20px;
+    left: 23px;
+    backdrop-filter: blur(4px);
+    background-color: #e5e5e5e6;
+    z-index: 999;
+    display: none;
+    width: 85%;
+    margin: 0 auto;
+    overflow-y: scroll;
+  }
+
+  .newElement {
+    background-color: #fff;
+    width: 90%;
+    padding: 30px 20px;
+    border-radius: 8px;
+    display: none;
+    font-family: "Poppins", sans-serif;
+    margin: 0 auto;
+  }
+
+  .flexText {
+    flex-direction: column;
+  }
+
+  .titleDisplay {
+    color: #172b4d;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+  }
```

Comment:
- [ ] Please fix the popup card styling in the desktop view by centering the positioning of the card first, then fix the widths, margins, and padding for all popup card elements, and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/162211548-b15c4f19-1bf1-46c1-a96d-4f5060fe90ee.png)  | ![image](https://user-images.githubusercontent.com/68796232/162211193-719cae90-8ef6-49fd-9057-a5af484e6258.png)  |